webrtc.html

```diff
@@ -1853,6 +1870,27 @@ <h4>Set the RTCSessionDescription</h4>
                       </li>
                     </ol>
                   </li>
+                  <li>
+                    <p>If we are processing a rollback then for each
```

Comment:
Note: I used the language "If we are processing a rollback" rather than 'If description is of type "rollback"' because until we fix #2332 by explicitly making rollback a separate part of steps, in this part of the algorithm the description variable has already been overwritten (e.g. it's type is not "rollback" even though we are processing a rollback). When all rollback steps are processed in a single place - I suggest here - then we can change this to 'If description is of type "rollback"'.

Reply:
PR #2384 fixes this, when rebasing these steps should be part of the steps introduced in that PR.